### ISSUE TITLE
[Suggestion] Add crafting recipes to upgrade Forges.

### ISSUE THREAD

**Issue by PlayfulPiano** [Wilyicaro/BetterFurnacesReforged#133]
### Better Furnaces Reforged Version

N/A

### Mod Loader Version

N/A

### Bug/Glitch Occurrence Context

Realized when looking through the mod's recipes that you can upgrade individual furnace tiers to forges, but without a method to uncraft, you're unable to upgrade those forges to higher tiers, as far as I can tell.
If it's not easy to setup a recipe for in-inventory upgrades, then maybe having the "shift right click" upgrades would give enough of the inventory space (probably putting the past "shift right click" upgrade in the recipe?).

### Crash Report

N/A

### What happened wrong?

N/A

**Comment by PlayfulPiano** [comment]
Guessing there's not a way to condense the recipes between the furnace → forge upgrade and the furnace tier upgrade?

**Comment by Wilyicaro** [comment]
I don't think so, because a forge upgrade needs to be more expensive than the furnace upgrade, and as a forge is better than 3 furnaces of the same tier, its upgrade obviously needs to be proportionally 3x more expensive than a furnace upgrade.

**Comment by Giovanni-Val** [comment]
why not just make a forge upgrade item that requires 3 furnace upgrades + a conductor block of the same tier? It's the easiest, most direct way to make it 3x as expensive while still mantaining some sense... it sucks that once you make a forge, you're stuck with it, or at least make downgrading the forge back into a furnace?
